src/Compilers/Core/Portable/CommandLine/TouchedFileLogger.cs

```diff
@@ -68,7 +66,7 @@ public void WriteReadPaths(TextWriter s)
             int i = 0;
             var readFiles = Interlocked.Exchange(
                 ref _readFiles,
-                null);
+                null!);
```

Comment:
Note the comment on this method when looking at the null suppressions here:

> After calling this method the logger is in an undefined state.

This seems to meet the `IDisosable` exception in annotations. Essentially the object is in an undefined state after this so the NRT state isn't important at this point cause any use of the object is already invalid.